_pages/security-events.md

```diff
@@ -20,25 +25,35 @@ sidenav:
 
 ## Getting started
 
-login.gov allows partners to notify us of various security-related events through our API.
+login.gov allows partners and Relying Parties (RPs) to notify us of various security-related events through our API.
 
 ### Auto-discovery
 
 login.gov provides a JSON endpoint for OpenID Connect auto-discovery at `/.well-known/risc-configuration`. In our agency integration environment, this is available at <https://idp.int.identitysandbox.gov/.well-known/risc-configuration>
 
 ## Supported Events
 
-login.gov supports a subset of the [OpenID RISC Event Types][openid-risc-events]:
+login.gov custom events, based on the [OpenID RISC Event Types][openid-risc-events], but not specific ones from that list at this time.
+
+### Authorization Fraud Detected
+
+RPs should submit this event when they believe a user's credentials have been compromised, that somebody who is not the user
+was able to sign in to a user's account. login.gov may force the reset of user's password when we receive this event.
 
+The **event_type** for this is:
 
-### Account Credential Change Required
+```
+https://schemas.openid.net/secevent/risc/event-type/login-gov/authorization-fraud-detected
+```
 
-Submit this event to login.gov if the user should change their credential (usually, reset their password).
+### Identity Fraud Detected
+
+RPs should submit this event when they believe a verified identity may be fraudulent. login.gov may reset the user's profile and verified attributes data when we receive this event.
```

Comment:
I didn't know a better way to say "proofed as somebody else" but open to wording/phrasing here

Reply:
My shot "RPs should submit this event when they suspect an account has been used to commit identity theft or related fraudulent activity."

Reply:
I like it! Thanks